### ISSUE TITLE
need mashapi api key

### ISSUE THREAD

**Issue by dvliman** [daryl/gomain#2]
ha looks like domainr is moving to mashapi to manage the authentication. Looks like we will need to update the code for this. Reading more into mashape..

**Comment by daryl** [comment]
Yeah, I got an email about this a couple weeks back. There's a free plan on Mashape, but you pay when you go over, it would have 10,000 requests a month limit, which I'm sure would be fine for this. I may do that, unless you know of any other services where I can request domain availability?

**Comment by kevinelliott** [comment]
Please do that!
